### DISCUSSION TITLE
Additional forecast JSON properties e.g. WBGT, apparent temp, wet bulb temp, wave height, tide

### DISCUSSION THREAD

**Discussion by stvnbryntksh** [weather-gov/api#662] [Q&A]
**Feature Request**: Some additional weather data in forecast calls  

Wet Bulb Globe Temperature (WBGT) is a specialized measurement I'd like to utilize but I don't see it being returned in forecast calls. I do see that some map layers may or may not return WBGT but no json api returns WBGT that I have found.

Wet Bulb (WB) Temp and apparent temp would also be helpful. These can be approximated or calculated but would be good to have in a standardized form from NWS.

Finally, marine data e.g. tides, waves etc would be quite useful. I realize there is another thread specifically about maritime data but I thought I'd add this here too.

Thanks

**Comment by sullynole** [comment]
WBGT is available if populated by local weather forecast offices, but not yet available everywhere.

National Ocean Service provides tides data in their web services: https://tidesandcurrents.noaa.gov/web_services_info.html

**Reply by stvnbryntksh** [reply]
Thanks for the quick reply. 

So to confirm, a call like: https://api.weather.gov/stations/[stationID]/observations/latest would contain a WBGT property if that particular station offers it?

Looking into NOAA, thanks. Also looked at NDFD. Any overview of the different services and their use cases would be appreciated.

**Reply by sullynole** [reply]
No, it's not available from station observations.  It would be in the forecastGridData url from the points endpoint, but I have not seen one populated yet.

More information on where to find WBGT here: https://www.weather.gov/media/notification/pdf2/scn22-43_wet_bulb_globe_temp_t2o_aaa.pdf

NDFD stitches together the weather forecast offices' grid domains into a single grid.  

API provides point access to the grids directly and provides text forecast summary of the grids, etc.  It also has NWS alerts and products; as well as station observations ingested from MADIS.  API does not yet handle marine forecast other than coastal waters raw data.  NWS does forecast wave height and wave period from the surface wind, which can be found in the API forecastGridData or the NDFD data.

**Reply by stvnbryntksh** [reply]
Thanks again, very helpful. I was looking at /forecast.

**Comment by rcboler** [comment]
Just bumping the request for **nationwide WBGT data**, or at least in the regions where heat is the most dangerous. 

I'm designing a heat safety app to popularize & explain WBGT to the general public. Trying to go live before another dangerous summer starts. Without WBGT data from the NWS, I've only found less-reliable, privately generated WBGT data feeds. Thanks y'all!

**Reply by sullynole** [reply]
Some offices are producing WBGT for api.weather.gov and it is found in the raw gridpoints endpoints.  For WBGT on a national level, that is currently available from NDFD grids. https://vlab.noaa.gov/web/mdl/ndfd

**Reply by rcboler** [reply]
@sullynole Thank you so much! Until your comment I couldn't publicly available WBGT data feed that didn't rely on scraping or something. Much obliged.
